core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -75,11 +77,12 @@ class AdminClientIntegrationTest extends KafkaServerTestHarness with Logging {
     super.tearDown()
   }
 
-  val brokerCount = 3
-  lazy val serverConfig = new Properties
+  val serverCount = 3
+  val consumerCount = 1
+  val producerCount = 1
 
   override def generateConfigs = {
-    val cfgs = TestUtils.createBrokerConfigs(brokerCount, zkConnect, interBrokerSecurityProtocol = Some(securityProtocol),
+    val cfgs = TestUtils.createBrokerConfigs(serverCount, zkConnect, interBrokerSecurityProtocol = Some(securityProtocol),
```

Comment:
Currently it's not needed even because the `serverConfig` is just empty. Maybe in the future some other tests will be added to this class configuring more server configs ?

Reply:
Makes sense, let's leave it here then.